### PR TITLE
fix:Removed unwanted apks in the apk branch

### LINKED ISSUE #1828: Remove Un needed apks from the apk branch
**Actual Behaviour**

Currently there are way too many apks in the apk branch.

**Expected Behaviour**

We need only 4-5 apks, one for each event + one that is built using repo's code.

### PR DESCRIPTION
**Pull request by shreyasnbhat**
Fixes #1828

### CODE REVIEW

**Comment by shreyasnbhat** [comment]
@the-dagger Please review.

File: upload-apk.sh [hunk 1]
```diff
@@ -11,22 +11,21 @@ git config --global user.email "noreply@travis.com"
 git config --global user.name "Travis-CI"
   
 git clone --quiet --branch=apk https://the-dagger:$GITHUB_API_KEY@github.com/fossasia/open-event-android  apk > /dev/null
-cd apk
```
Comment (the-dagger) [inline upload-apk.sh]: You are cloning the app branch in a folder named app, so you need to cd to that folder too.

**Review by the-dagger — CHANGES_REQUESTED** [review]

**Comment by shreyasnbhat** [comment]
@the-dagger Updated.

**Review by the-dagger — APPROVED** [review]

**Comment by the-dagger** [comment]
@shreyasnbhat This isn't working.
Please try the script on your fork once again and send a PR with the fix.

**Comment by shreyasnbhat** [comment]
@the-dagger Having a look now.

**Comment by shreyasnbhat** [comment]
@the-dagger figured out the error.
`git add -f .` doesn't track deleted files. We had to use `git add -f -A` to track the deleted files

**Pull request merged**